charts/console/values.yaml

```diff
@@ -247,10 +239,9 @@ secret:
 
 # -- Settings for license key, as an alternative to secret.enterprise when
 # a license secret is available
-enterprise:
-  licenseSecretRef:
-    name: ""
-    key: ""
+licenseSecretRef:
```

Comment:
As mentioned above, it's preferable to default this to `null` rather than a full object with zero valued keys. That way you can check `licenseSecretRef != nil` instead of `licenseSecretRef.Name != "" && licenseSecretRef.Key != ""`